Tensile/ClientWriter.py

```diff
@@ -1192,7 +1198,7 @@ def writeClientParameters(forBenchmark, solutions, problemSizes, stepName, \
           solution["AssertFree0ElementMultiple"],
           solution["AssertFree1ElementMultiple"],
           solution["AssertMinApproxSize"],
-          "true" if solution["LdcEqualsLdd"] else "false",
+          "false",
```

Comment:
Is this invalid if globalParameters["CEqualD"] is true ? Do we need a sanity check here?

Reply:
we will abort training when CEqualD is true and stride is not equal between C and D.